### ISSUE TITLE
Learning Sim-to-Real Dense Object Descriptors for Robotic Manipulation. (arXiv:2304.08703v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#25677]
Learning Sim-to-Real Dense Object Descriptors for Robotic Manipulation. (arXiv:2304.08703v1 [cs.RO])<br>
https://ift.tt/tkhycE4<br>
<p>It is crucial to address the following issues for ubiquitous robotics manipulation applications: (a) vision-based manipulation tasks require the robot to visually learn and understand the object with rich information like dense object descriptors; and (b) sim-to-real transfer in robotics aims to close the gap between simulated and real data. In this paper, we present Sim-to-Real Dense Object Nets (SRDONs), a dense object descriptor that not only understands the object via appropriate representation but also maps simulated and real data to a unified feature space with pixel consistency. We proposed an object-to-object matching method for image pairs from different scenes and different domains. This method helps reduce the effort of training data from real-world by taking advantage of public datasets, such as GraspNet. With sim-to-real object representation consistency, our SRDONs can serve as a building block for a variety of sim-to-real manipulation tasks. We demonstrate in experiments that pre-trained SRDONs significantly improve performances on unseen objects and unseen visual environments for various robotic tasks with zero real-world training.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
